cmd/geoipupdate/main.go

```diff
@@ -40,10 +40,11 @@ func main() {
 
 	config, err := geoipupdate.NewConfig(args.ConfigFile, defaultDatabaseDirectory, args.DatabaseDirectory, args.Verbose)
 	if err != nil {
-		fatalLogger("error loading configuration file", err)
+		fatalLogger(fmt.Sprintf("error loading configuration file %s", args.ConfigFile), err)
 	}
+
 	if config.Verbose {
-		log.Printf("Using config file %s", args.ConfigFile)
+	        log.Printf("Using config file %s", args.ConfigFile)
```

Comment:
Could you please rebase out these whitespace changes?  At least the one on line 47 which `gofmt` undoes.  You can run it on a single file with `gofmt -w ./cmd/geoipupdate/main.go`.

Reply:
Done.